### PR TITLE
Redmine 3.4

### PR DESCRIPTION
**Pull request by taqueci**
Hi, @mrliptontea

I have updated files for Redmine 3.4.
Could you please review and merge?

This pull request resolves the following issues:
* [1474 Show last comment/notes in the issue list](http://www.redmine.org/issues/1474)
* [6375 Last updated by colum in issue list](http://www.redmine.org/issues/6375)
* [15361 Use css pseudo-classes instead of cycle("odd", "even")](http://www.redmine.org/issues/15361)
* [25297 In place editing of "My page" layout](http://www.redmine.org/issues/25297)
* [25515 View attachments on the issue list](http://www.redmine.org/issues/25515)
* [25775 Show assignee's icon in addition to author's icon](http://www.redmine.org/issues/25775)
* [25999 View repository content by default (instead of the history)](http://www.redmine.org/issues/25999)

### CODE REVIEW

File: src/javascripts/modules/SidebarToggler.js [hunk 1]
```diff
@@ -38,7 +38,9 @@ PurpleMine.SidebarToggler = (function () {
 
     // Fix issue with context menu position
     if (this.$main.css('position') === 'relative') {
-      $('#context-menu').appendTo('#wrapper3')
+      $(window).load(function () {
+        $('#context-menu').appendTo('#wrapper3')
+      })
```
Comment (mrliptontea) [inline src/javascripts/modules/SidebarToggler.js]: Were there issues with loading this? All JS plugins are loaded on .ready in [src/javascripts/theme.js](/mrliptontea/PurpleMine2/blob/399b5abeb9f2e6a73cce70aec1f6796b59d66da1/src/javascripts/theme.js), so it shouldn't be necessary. 🤔

**Comment by mrliptontea** [comment]
Except for one question all looking good. 👍

@taqueci would you be interested in becoming this theme's maintainer? 😃 I'm no longer Redmine user, so I'm not up to date with changes.

**Comment by mkleins** [comment]
:+1: for merge

**Comment by taqueci** [comment]
Hi, @mrliptontea

> Were there issues with loading this? 

Context menu initialization order is changed by [r15936](http://www.redmine.org/projects/redmine/repository/revisions/15936).
Function contextMenumInit() is called:
* before theme.js (called at the bottom of HTML) (Redmine 3.3)
* after .ready of theme.js (Redmine 3.4)

> would you be interested in becoming this theme's maintainer?

I like PurpleMine2 theme! :smiley:
However, I don't have enough skill for the maintainer...

**Comment by mrliptontea** [comment]
@taqueci I could add you as a project member. Think about it. For now, thank you for PR

**Pull request merged**